src/sentry/south_migrations/0501_auto__add_index_group_project_id.py

```diff
@@ -0,0 +1,1491 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
```

Comment:
@JTCunning  Do you want to this offline or online ?

Reply:
This has to be offline too. It'll take way too long to run.